worker/src/index.ts

```diff
@@ -51,6 +52,18 @@ function createTarballWorkerPool (): WorkerPool {
   return workerPool
 }
 
+function calcMaxWorkers () {
+  if (process.env.PNPM_WORKERS) {
```

Comment:
What does this env variable do? Couldn't find any docs related to it

Reply:
It isn't intended to be used by users. It tells how many CPUs pnpm should leave idle.